### PR TITLE
Task/update readme docker run

### PR DESCRIPTION
**Pull request by PritchardJasonR**
add docker compose run example
Add documentation for backend yarn seed and some examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +73,31 @@ To run the application, you can use the following scripts:
...
+    yarn seed
+    yarn start:dev
+```
+
+**Seeding For Tests**
+
+Resetting Data Before Tests: If you have end-to-end tests or integration tests, you might want to ensure a consistent database state before running them. To reset and seed your database:
+
+```bash
+yarn migrate:run
```
Comment (d4search) [inline README.md:96]: Identation

File: README.md [hunk 2]
```diff
@@ -66,6 +73,31 @@ To run the application, you can use the following scripts:
...
+Seeding allows you to populate your database with initial data. This can be useful for testing or when you want to ensure a consistent set of data is available across different environments.
+
+```yarn seed```: Builds and then runs the seeder.js file from the dist/src directory. 
+
+
+**Seeding for Development**
+
+Before running the development server, you might want to seed your database with some test data.
+```bash
+    yarn seed
```
Comment (d4search) [inline README.md:87]: yarn seed isn't the complete command

its `yarn seed -seeder UserSeeder,ServiceSeeder,ServiceSlotSeeder`

File: README.md (hunk 1 above)
Comment (PritchardJasonR) [reply README.md]: resolved in the commit [respond to comments](https://github.com/SanctuaryTeam/diablo4trading-be/pull/54/commits/360003922fae33b46f82afe035df7c8c78a4e183)

**Review by d4search — APPROVED** [review]

**Review by Sparky-Kitty — APPROVED** [review]
Nice documentation. Thanks!

File: README.md [hunk 3]
```diff
@@ -66,6 +73,33 @@ To run the application, you can use the following scripts:
...
+
+_Note_: **CAUTION** when using seed scripts, especially in production environments. These scripts can replace existing data, potentially leading to data loss or inconsistencies.
+
+**Seeding For Tests**
+
+Resetting Data Before Tests: If you have end-to-end tests or integration tests, you might want to ensure a consistent database state before running them. To reset and seed your database:
+
+```bash
+ yarn migrate:run
+ yarn seed -seeder UserSeeder,ServiceSeeder,ServiceSlotSeeder
```
Comment (d4search) [inline README.md:99]: Should be `--`

File: README.md [hunk 4]
```diff
@@ -66,6 +73,33 @@ To run the application, you can use the following scripts:
...
+Seeding allows you to populate your database with initial data. This can be useful for testing or when you want to ensure a consistent set of data is available across different environments.
+
+```yarn seed```: Builds and then runs the seeder.js file from the dist/src directory. 
+
+
+**Seeding for Development**
+
+Before running the development server, you might want to seed your database with some test data.
+```bash
+ yarn seed -seeder UserSeeder,ServiceSeeder,ServiceSlotSeeder
```
Comment (d4search) [inline README.md:87]: Should be `--`

**Pull request merged**